example-app/app/layout.tsx

```diff
@@ -1,4 +1,5 @@
 import './globals.css';
+import { getSession } from '@auth0/nextjs-auth0';
```

Comment:
Hi @seanparmelee - we're gonna have to remove this from the sample app because it's making the Edge routes fail (`'@auth0/nextjs-auth0'` can only be imported on the Node runtime and `layout.tsx` is used in Node and Edge routes)